src/main/java/org/shanerx/tradeshop/utils/relativedirection/LocationOffset.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ *                         Copyright (c) 2016-2019
```

Comment:
We need to update this :S

Reply:
Yeah I keep thinking about it and forgetting